task/README.md

```diff
@@ -0,0 +1,143 @@
+# build-dm-verity-image Task — README
+
+This README describes the Tekton Task defined in `build-dm-verity-image.yaml` and
+how it fits into the repository pipeline for building a DM‑Verity PodVM image.
+
+## Purpose and constraints
+
+The `build-dm-verity-image` Task automates building a disk image (QCOW2) with
+DM‑Verity support and packaging it as an OCI artifact, as part of our Konflux CI.
```

Comment:
It's not an OCI artifact yet, but a OCI image with the qcow2 file inside. This depends on what format the OSC operator expects. It's a good idea to change it to artifact.